### DISCUSSION TITLE
Signup list feedback

### DISCUSSION THREAD

**Discussion by bf-mike-mkt** [betfinio/public#47] [Bugs and Reports]
1. Member sorting (time signup) is missing. Default should be newest first. That is the default sorting scheme. 
2. LVL of user should be possible to switch between binary / linear. Linear should be the default view.
3. Binary side does not work in all cases - some of them are blank. 
4. Total volume filter displayed last page with numbers after zero results.
<img width="263" alt="image" src="https://github.com/user-attachments/assets/1ac16e5d-f5f9-42b7-b9b0-0d6266f0aebf">
5. Once address is filtered, there shuld be option to unfilter it easily

**Comment by bf-monster** [comment]
closing as partially resolved
